modules/programs/git.nix

```
@@ -274,6 +300,11 @@ in {
   config = mkIf cfg.enable (mkMerge [
     {
       home.packages = [ cfg.package ];
+      assertions = [{
+        assertion = !(cfg.delta.enable && cfg.difftastic.enable);
+        message =
+          "Only one of delta or difftastic can be enabled at the same time.";
```

I think we should specify the full option paths here.
```suggestion
          "Only one of 'git.delta.enable` or `git.delta.difftastic.enable` can be set to true at the same time.";
```